src/StoreKit/SKCloudServiceSetupOptions.cs

```diff
@@ -10,10 +12,12 @@ partial class SKCloudServiceSetupOptions {
 
 		public virtual SKCloudServiceSetupAction? Action {
 			get {
-				return (SKCloudServiceSetupAction?) (SKCloudServiceSetupActionExtensions.GetValue (_Action));
+				if (_Action is null)
+					ObjCRuntime.ThrowHelper.ThrowArgumentNullException (nameof (_Action));
```

Comment:
SKCloudServiceSetupAction? allows to return a null, just return null if _Action is null piece of 🍰 :)

Reply:
Agreed, just return `null` here instead, and no exception will need to be thrown.